driver/src/contract.rs

```diff
@@ -90,7 +89,7 @@ impl SnappContract for SnappContractImpl {
             .and_then(|block_option| {
                 match block_option {
                     Some(block) => Ok(block.timestamp),
-                    None => Err(web3::Error::from("Current block not found"))
+                    None => Err(web3::error::Error::Decoder(String::from("Current block not found")))
```

Comment:
Is this the new error you were looking for!?

Reply:
no ...

Reply:
it looks like it's coming from web3...